lib/bolt/config/options.rb

```diff
@@ -494,7 +432,7 @@ module Options
                        "specified in the URI.",
           type: String,
           enum: TRANSPORT_CONFIG.keys,
-          _plugin: false,
+          _plugin: true,
```

Comment:
Was this just a bug?

Reply:
I believe so. I don't think we wanted to disallow plugins for the transport.